1-Draft/RFCNNNN-AdditionalTelemetry.md

```diff
@@ -0,0 +1,170 @@
+---
+RFC: RFC
+Author: Sydney Smith
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Telemetry
+Comments Due: 3/29/19
+Plan to implement: <Yes | No>
+---
+
+# Additional Telemetry in PowerShell
+
+Add usage telemetry across PSEdition, Platform, Application, session type and command types.
+Improve the opt-out mechanisms for PowerShell telemetry.Provide community access to aggregated
+statistics.
+
+## Motivation
+
+To make the best possible investments in PowerShell we need answers backed by data to the following questions:
+
+- Is PowerShell Core usage growing (in terms of number of starts)?
+- Is the PowerShell Core user-base growing (both in terms of number of users and devices)?
+- How is PowerShell being used? What is the usage distribution across command types and session type?
+- What are impediments to PowerShell Core usage growth?
+- What are issues that customers are hitting in PowerShell Core?

```

Comment:
🙋‍♂️ How are the metrics mentioned in this document going to help to answer the above two questions?

Reply:
Great question! The idea is that by understanding the distribution across command types we might be able to better diagnose where issues are occurring if usage deviates. For example if we understand what "typical" cmd type distribution looks like and we notice a change, we might be able to identify a bug. These metrics become more interesting in looking at how they change overtime and how they change in response to changes we make to Powershell Core.